media-gfx/openvdb/openvdb-4.0.2-r3.ebuild

```diff
@@ -34,7 +38,7 @@ RDEPEND="
 	python? (
 		${PYTHON_DEPS}
 		$(python_gen_cond_dep '
-			>=dev-libs/boost-1.62:=[python,${PYTHON_MULTI_USEDEP}]
+			dev-libs/boost:=[python?,${PYTHON_MULTI_USEDEP}]
 			dev-python/numpy[${PYTHON_MULTI_USEDEP}]
 		')
 	)"
```

Comment:
Can't get the comment on the right line but pkgconfig, the doc deps, and possibly cppunit should be in BDEPEND.

Reply:
I have created a BDEPEND section for pkgconfig, the doc deps, cppunit and cmake (where required) for all the openvdb ebuilds.